index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin<br />Glad to see<br />
         </li>
     </ul>
 </div>
-</section>
\ No newline at end of file
+</section>
+<!--Works Section page 1-->
+<section class="page1">
```

Comment:
As the previous reviewer suggested, Kindly use `display:grid` for designing this section. :+1: